src/Evolution/Systems/ScalarTensor/StressEnergy.hpp

```diff
@@ -3,4 +3,106 @@
 
 #pragma once
 
-namespace ScalarTensor {}  // namespace ScalarTensor
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "Evolution/Systems/ScalarTensor/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace ScalarTensor {
+
+/*!
+ * \brief Add in the trace-reversed stress-energy source term to the \f$\Pi\f$
+ * evolved variable of the ::gh system.
+ *
+ * \details The only stress energy source term in the Generalized Harmonic
+ * evolution equations is in the equation for \f$\Pi_{a b}\f$:
+ * \f[
+ * \partial_t \Pi_{ab} + \text{\{spatial derivative terms\}} =
+ * \text{\{GH source terms\}}
+ * - 16 \pi \alpha (T^{(\Psi)}_{ab} - \frac{1}{2} g_{a b} g^{cd}T^{(\Psi)}_{cd})
+ * ~.
+ * \f]
+ *
+ * (note that this function takes as argument the trace-reversed stress-energy
+ * tensor)
+ *
+ * This function adds that contribution to the existing value of `dt_pi`. The
+ * spacetime terms in the GH equation should be computed before passing the
+ * `dt_pi` to this function for updating.
+ *
+ * \param dt_pi Time derivative of the $\Pi_{ab}$ variable in the ::gh system.
+ * The vacuum part should be computed before with ::gh::TimeDerivative
+ * \param trace_reversed_stress_energy Trace-reversed stress energy tensor of
+ * the scalar $T_{ab}^{(\Psi)}$.
```

Comment:
`_\mathrm{TR}` is missing I think? Maybe write out the definition as well here.

Reply:
Oh, right. Fixed.